File: api/v2010/address/create-example-1/create-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var address = AddressResource.Create(
            friendlyName: "Billing - Customer 123",
            city: "Berlin",
            customerName: "Customer 123",
            isoCountry: "DE",
            postalCode: "10875",
            region: "Berlin",
            street: "1 Hasselhoff Lane",
            pathAccountSid: "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(address.Sid);
    }
}

File: api/v2010/address/create-example-1/create-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

address = @client.addresses.create(
                              friendly_name: 'Billing - Customer 123',
                              city: 'Berlin',
                              customer_name: 'Customer 123',
                              iso_country: 'DE',
                              postal_code: '10875',
                              region: 'Berlin',
                              street: '1 Hasselhoff Lane'
                            )

puts address.sid
